src/LEDMatrixDriver.cpp

```diff
@@ -160,13 +160,32 @@ void LEDMatrixDriver::_sendCommand(uint16_t command)
 	SPI.endTransaction();
 }
 
+uint8_t LEDMatrixDriver::_reverseBits(uint8_t n)
+{
+	uint8_t x;
+	for(auto i = 7; n; ) {
+		x |= (n & 1) << i;
+		n >>= 1;
+		-- i;
+	}
+	return x;
+}
+	
 void LEDMatrixDriver::_displayRow(uint8_t row)
 {
 	SPI.beginTransaction(spiSettings);
 	digitalWrite(ssPin, 0);
 	for (uint16_t d = 0; d < N; d++)
 	{
-		uint16_t cmd = ((row + 1) << 8) | frameBuffer[d + row*N];
+		uint8_t col = frameBuffer[d + row*N];
+
+		if ((moduleFlip & static_cast<int>(flipDirection::flipColumn)) != 0)

```

Comment:
How about caching if it's reversed or not and not checking it every time?
You can do it once per function instead of doing it for every element.